ports/raspberrypi/boards/cytron_edu_pico/mpconfigboard.h

```diff
@@ -0,0 +1,10 @@
+#define MICROPY_HW_BOARD_NAME "Cytron EDU PICO"
+#define MICROPY_HW_MCU_NAME "rp2040"
+
+#define CIRCUITPY_DIGITALIO_HAVE_INVALID_PULL (1)
+#define CIRCUITPY_DIGITALIO_HAVE_INVALID_DRIVE_MODE (1)
+
+#define MICROPY_HW_LED_STATUS   (&pin_CYW0)
+
+#define CIRCUITPY_BOARD_I2C         (1)
+#define CIRCUITPY_BOARD_I2C_PIN     {{.scl = &pin_GPIO5, .sda = &pin_GPIO4}}
```

Comment:
The Grove port table in the datasheet indicates SPI pins for ports 1-6, but I'm thinking those devices don't actually communicate over SPI which would mean the only predefined SPI would be for the SD card which you might want to define as board.SPI
```
#define CIRCUITPY_BOARD_SPI         (1)
#define CIRCUITPY_BOARD_SPI_PIN     {{.clock = &pin_GPIO18, .mosi = &pin_GPIO19, .miso = &pin_GPIO16}}
```